pyramid3.cpp

```diff
@@ -1,21 +1,20 @@
-#include<stdio.h>
-int main()
-{
+#include <stdio.h>
+
+int main(){
 	int i, j;
-	char input, alphabet='A';
+	char input, alphabet = 'A';
 	printf("Enter the uppercase character you want to print in last row:");
-	scanf("%c",&input);
-	for(i=1; i<=(input-'A'+1); ++i)
-{
-
-	for(j=1; j<=i; ++j)
+	scanf("%c", &input);
+	for (i = 1; i <= (input - 'A' + 1); ++i)
 	{
-	printf("%c",alphabet);	
-	}
-	printf("\n");
-}
-++alphabet;
-return 0;
 
+		for (j = 1; j <= i; ++j)
+		{
+			printf("%c", alphabet);
+		}
+		printf("\n");
+	}
+	++alphabet;
```

Comment:
avoid `++` before variable whenever possible.
Always try to use `i++` instead of `++i`.